src/client/pages/SignInSuccess.tsx

```diff
@@ -1,233 +1,99 @@
 import React, { useContext } from 'react';
 import { css } from '@emotion/react';
-import {
-  body,
-  brand,
-  from,
-  headline,
-  neutral,
-  space,
-  titlepiece,
-  until,
-} from '@guardian/source-foundations';
-import { Button, Checkbox } from '@guardian/source-react-components';
-
+import { from, headline, until } from '@guardian/source-foundations';
+import { Button } from '@guardian/source-react-components';
+import { getAutoRow, gridItemColumnConsents } from '@/client/styles/Grid';
 import { Header } from '@/client/components/Header';
 import { Footer } from '@/client/components/Footer';
 import { GeoLocation } from '@/shared/model/Geolocation';
 import { Consent } from '@/shared/model/Consent';
-import { buildUrlWithQueryParams } from '@/shared/lib/routeUtils';
+import { CommunicationCard } from '@/client/components/CommunicationCard';
+import { CONSENT_IMAGES } from '@/client/models/ConsentImages';
 import { CsrfFormField } from '@/client/components/CsrfFormField';
+import { buildUrlWithQueryParams } from '@/shared/lib/routeUtils';
 import { onboardingFormSubmitOphanTracking } from '@/client/lib/consentsTracking';
 import { ClientState } from '@/shared/model/ClientState';
 import { ClientStateContext } from '@/client/components/ClientState';
-import {
-  gridItem,
-  gridRow,
-  gridRowPadding,
-  innerGridRow,
-} from '@/client/styles/Grid';
-import { EnvelopeImage } from '@/client/components/EnvelopeImage';
-
-const titleSpanDefinition = {
-  DESKTOP: { start: 2, span: 10 },
-  LEFT_COL: { start: 2, span: 11 },
-  WIDE: { start: 2, span: 11 },
-};
-
-const envelopeSpanDefinition = {
-  DESKTOP: { start: 9, span: 3 },
-  LEFT_COL: { start: 11, span: 3 },
-  WIDE: { start: 13, span: 3 },
-};
-
-const formSpanDefinition = {
-  DESKTOP: { start: 2, span: 6 },
-  LEFT_COL: { start: 2, span: 6 },
-  WIDE: { start: 2, span: 6 },
-};
-
-const titleWrapper = css`
-  background: ${brand[400]};
-  border-top: 1px solid ${brand[600]};
-
-  ${from.desktop} {
-    border-top: none;
-    margin-top: 1px;
-  }
-`;
-
-const autoMargin = css`
-  margin: 0 auto;
-`;
-
-const title = css`
-  ${titlepiece.small()}
-  ${gridItem(titleSpanDefinition)}
-  font-size: 36px;
-  color: ${neutral[100]};
-  margin: 60px 0 70px;
-
-  ${from.desktop} {
-    ${titlepiece.large()}
-    margin: 60px 0 130px;
-  }
-`;
+import { ConsentsContent, controls } from '@/client/layouts/shared/Consents';
+import { mainStyles } from '@/client/layouts/ConsentsLayout';
+import { ConsentsBlueBackground } from '@/client/components/ConsentsBlueBackground';
 
-const formStyles = css`
-  ${gridItem(formSpanDefinition)}
+const heading = css`
+  ${headline.small({ fontWeight: 'bold' })}
 
-  ${until.desktop} {
-    padding: 0 ${space[5]}px;
+  ${from.tablet} {
+    font-size: 32px;
```

Comment:
shame we can't use font sizes from the Source typography scale here. I think there's a 34 but no 32

Reply:
I should mention that `src/client/styles/Grid.ts` is actually completely unchanged now that I'm using the correct design.